worker/src/lib.rs

```diff
@@ -36,6 +36,7 @@ pub use crate::formdata::*;
 pub use crate::global::Fetch;
 pub use crate::headers::Headers;
 pub use crate::http::Method;
+pub use crate::queue::*;
```

Comment:
With queues being in beta I think we should at least gate it behind a feature flag.

Reply:
Good idea, I've added a new 'queue' feature and updated the readme.